File: SamplePlannerWith20V38E/Count.java
import java.util.Scanner;

public class Count
{
	public static void main(String[] args)
	{
		Scanner in = new Scanner(System.in);
		int[] array = new int[3];

		while(in.hasNext())
		{
			int temp = in.nextInt();
			array[temp]++;
		}

		for(int i = 0; i < 3; i++)
			System.out.println(array[i]);
	}
}

File: SamplePlannerWith20V38E/count.c
#include <stdio.h>

int main()
{
	int i = 0;
	int temp = 0;
	int array[3];
	for(i = 0;i < 3; i++)
		array[i] = 0;
	while(i < 20)
	{
		scanf("%d", &temp);
		printf("%d\n",temp);
		array[temp]++;
		i++;
	}
	for(i = 0;i < 3; i++)
		printf("%d\t%d\n", i, array[i]);

}